readme.rst

```diff
@@ -186,14 +186,13 @@ following line to ``.wokeignore``:
 Change checked file-types and locations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, only reST files are checked for inclusive language --- and only
+By default, only reST files are checked for inclusive language — and only
 those in ``docs/`` and its subdirectories. To check Markdown files for example,
```

Comment:
This is actually not true - there's no ``docs/`` in the command. What we mean is that it runs on the files in the documentation folder (which might be ``.`` or ``docs/`` or ``whatever`` depending on how the repo is set up).

Reply:
Good point, Ruth - thanks. Fixed now I think.